components/Navbar.vue

```diff
@@ -254,10 +254,6 @@ export default class NavbarMenu extends mixins(
     return this.$store.getters['history/getCurrentlyViewedItem']?.name || ''
   }
 
-  get isRedesignedLandingPage() {
-    return this.$route.name === 'index' && this.redesign
-  }
```

Comment:
Hey, pls look carefully. The condition is not only `isRedesign`.
So, you could not remove this function and also the search bar component.

```ts
get isLandingPage() {
     return this.$route.name === 'index'
}
```

Reply:
oops, I missed this one